src/api/frame/frame.ts

```diff
@@ -1,3 +1,4 @@
+declare var fin: any;
```

Comment:
Hipster nitpick: `var` is so 2014... cool kids use `const`

Not actually a blocker obviously.